addons/comments/initSettings.inc.sqf

```diff
@@ -0,0 +1,70 @@
+[
+    QGVAR(enableComments),
```

Comment:
`QGVAR(comment)`

Reply:
This setting does not prevent placing comments. It only determines if to show or to hide them. So maybe `QGVAR(showComments)`?

Reply:
I think the setting should be "enabled" or not. To me, it does not make sense to be able to place comments if I cannot see them.